cloud-sql/sql-server/client-side-encryption/noxfile_config.py

```diff
@@ -25,7 +25,7 @@
     "ignored_versions": ["2.7", "3.6"],
     # Old samples are opted out of enforcing Python type hints
     # All new samples should feature them
-    "enforce_type_hints": True,
+    "enforce_type_hints": False,
```

Comment:
```suggestion
    "enforce_type_hints": True,
```

I meant to do this in Dataflow not here 😁